Source/Core/DolphinQt2/Translate.h

```diff
@@ -0,0 +1,261 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QString>
+
+
+class Translate final
```

Comment:
Since all members of this class are static, the constructor should be made private to prevent instantiation of the class.